### PR TITLE
8225045: javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java fails on linux-x64

### PR DESCRIPTION
**Pull request by prsadhuk**
This test was failing on mach5 nightly on ubuntu systems long time back. Modified the test to add some delays and call robot.waitForIdle() to make it more stable and the resultant test passes on all mach5 systems including linux consisting of ubuntu 19.10, 18.04,20.10,20.04 so we can remove the test from problemlist.
mach5 job running for several iterations is posted in JBS.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8225045](https://bugs.openjdk.java.net/browse/JDK-8225045): javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java fails on linux-x64

### Reviewers
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)
 * [Pankaj Bansal](https://openjdk.java.net/census#pbansal) (@pankaj-bansal - **Reviewer**)
 * [Tejpal Rebari](https://openjdk.java.net/census#trebari) (@trebari - Committer)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/2003/head:pull/2003`
`$ git checkout pull/2003`

### CODE REVIEW

File: test/jdk/javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java [hunk 1]
```diff
@@ -61,7 +63,7 @@
 
     public static void main(String[] args) throws Exception {
         robot = new Robot();
-        robot.delay(2000);
+        robot.setAutoDelay(100);
```
Comment (mrserb) [inline test/jdk/javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java:66]: Why do you need an autodeley here? It seems the test does not use any of mouseXXX/keyXXX methods.

File: test/jdk/javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java [hunk 2]
```diff
@@ -208,6 +216,8 @@ private static void getIconBufferedImage() throws Exception {
     private static void testIfSame(final String lookAndFeelString)
             throws Exception {
         if (!bufferedImagesEqual(imageIconImage, iconImage)) {
+            ImageIO.write(imageIconImage, "png", new File("imageicon-fail.png"));
+            ImageIO.write(iconImage, "png", new File("iconImage-fail.png"));
```
Comment (trebari) [inline test/jdk/javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java:220]: Is this for debugging purpose when the test fails ?

**Review by mrserb — APPROVED** [review]

**Review by pankaj-bansal — APPROVED** [review]

**Review by trebari — APPROVED** [review]

File: test/jdk/javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java (hunk 2 above)
Comment (prsadhuk) [reply test/jdk/javax/swing/JInternalFrame/8146321/JInternalFrameIconTest.java:220]: yes

**Pull request closed without merging**